k2/csrc/benchmark/array_ops_benchmark.cu

```diff
@@ -0,0 +1,64 @@
+/**
+ * @brief Benchmarks for array_ops.
+ *
+ * @copyright
+ * Copyright (c)  2020  Xiaomi Corporation (authors: Fangjun Kuang)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ */
+
+#include <cstdlib>
+
+#include "k2/csrc/array_ops.h"
+#include "k2/csrc/benchmark/benchmark.h"
+
+namespace k2 {
+
+template <typename T>
+static BenchmarkStat BenchmarkExclusiveSum(int32_t dim) {
+  ContextPtr context = GetCudaContext();
```

Comment:
Don't we want to do the test for both Cuda and CPU?
Of course we could easily do this in separate processes (e.g. separate invocations of the benchmark program) which would help speed... dunno how easy it would be to set up and what the others think...

Reply:
Support for CPU is also added.

The current output format is CSV, which I think is very easy to parse. We can also output a format
that is compatible with google/benchmark. Don't know if you guys have some suggestions here.